### ISSUE TITLE
php 7.1 error in Eye Form

### ISSUE THREAD

**Issue by bradymiller** [openemr/openemr#1158]
Used the following demo to test this:
http://www.open-emr.org/wiki/index.php/Development_Demo#Alternate_Demo_.28Alpine_3.6_-_uses_php_7.1.29

When trying to open the Eye Form, get a white screen of death with following output in error log:
``````
[27-Oct-2017 17:51:17 UTC] PHP Warning:  Illegal string offset 'marital_status' in /var/www/localhost/htdocs/openemr/interface/forms/eye_mag/php/eye_mag_functions.php on line 1808
[27-Oct-2017 17:51:17 UTC] PHP Fatal error:  Uncaught Error: Cannot use string offset as an array in /var/www/localhost/htdocs/openemr/interface/forms/eye_mag/php/eye_mag_functions.php:1808
Stack trace:
#0 /var/www/localhost/htdocs/openemr/interface/forms/eye_mag/view.php(101): build_PMSFH('1')
#1 /var/www/localhost/htdocs/openemr/interface/patient_file/encounter/view_form.php(16): include_once('/var/www/localh...')
#2 {main}
  thrown in /var/www/localhost/htdocs/openemr/interface/forms/eye_mag/php/eye_mag_functions.php on line 1808
``````

**Comment by sjpadgett** [comment]
@bradymiller I don't see this error after issue #1215 fix also I use PHP Version 7.1.7RC1. Looks like now you must define array depth but just how far I don't know.

**Comment by bradymiller** [comment]
Also confirmed that this error has resolved with your fixes on the PHP 7.1 demo:
http://www.open-emr.org/wiki/index.php/Development_Demo#Alternate_Demo_.28Alpine_3.6_-_uses_php_7.1.29

**Comment by bradymiller** [comment]
fixed by @sjpadgett
